src/ItemBadges/ItemBadges.tsx

```diff
@@ -44,10 +52,14 @@ const ItemBadges = ({
   isPublishedTooltip = 'Published',
   isPublic = false,
   isPublicTooltip = 'Public',
+  isCollapsible = false,
+  isCollapsibleTooltip = 'Collapsible',
+  showChatbox = false,
+  showChatboxTooltip = 'Chat',
   backgroundColor = 'grey.600',
 }: Props): JSX.Element => {
   return (
-    <AvatarGroup>
+    <AvatarGroup max={10}>
```

Comment:
Info: do we use this `max` prop as a "safety" to display properly?

Reply:
Yes I set it to an arbitrary number higher than the number of badges we use, the default in MUI is `5` after which the avatars are grouped and displayed as (+X)